axiom/nr_txn.c

```diff
@@ -2582,10 +2588,16 @@ nrobj_t* nr_txn_event_intrinsics(const nrtxn_t* txn) {
       params, txn->unscoped_metrics, "WebFrontend/QueueTime", "queueDuration");
   nr_txn_add_metric_total_as_attribute(params, txn->unscoped_metrics,
                                        "External/all", "externalDuration");
+  nr_txn_add_metric_count_as_attribute(params, txn->unscoped_metrics,
```

Comment:
Does this affect any integration/multiverse tests or do they all wildcard the External/all count so this has never been tested as a fixed value?

Reply:
This is superceded by the https://github.com/newrelic/newrelic-php-agent/pull/1006 PR; once this is rebased on that, the externalCallCount changes will go away.  But this doesn't require any changes in multiverse (I ran both this messagesegment branch and the sqs instrumentation branch against multiverse).